docs/guide/setup/creating_atom_mappings_and_scores.rst

```diff
@@ -74,11 +67,9 @@ are coloured blue.
    :alt: Sample output of 2d mapping visualisation
 
 
-These 2D mappings can be saved to file using the :func:`LigandAtomMapping.draw_to_file` function.
+These 2D mappings can be saved to file using the :func:`LigandAtomMapping.draw_to_file()` method.
 
-With the ``py3dmol`` package installed,
-we can also view the mapping in 3D allowing us to manually inspect the spatial overlap
-of the mapping.
+With the ``py3dmol`` package installed, we can inspect the spatial overlap of the mapping in 3D.
```

Comment:
Right now we have a "batteries included approach" and `py3dmol` is a runtime dep in the conda-forge pacakge https://github.com/conda-forge/openfe-feedstock/blob/main/recipe/meta.yaml#L43

So I think we can do  
```suggestion
With the ``py3dmol`` package, we can inspect the spatial overlap of the mapping in 3D.
```

or something like that